app/models/concerns/course/reminder_concern.rb

```diff
@@ -57,4 +59,14 @@ def setup_closing_reminders
       end
     end
   end
+
+  # Determines whether the opening reminder should be sent. Reminders always should be sent unless
+  # the start_at and the old start_at dates are both in the past.
+  #
+  # @return [Boolean] True if an opening reminder should be sent
+  def should_send_opening_reminder
+    time_now = Time.zone.now
+    return false if start_at < time_now && start_at_was < time_now
```

Comment:
Don't we usually avoid returning `false` and `true` explicitly with the `if` statement?

Or is this for readability? Instead of
`!(start_at < time_now && start_at_was < time_now)`